flake.nix

```diff
@@ -28,9 +28,11 @@
         in rustPlatform.buildRustPackage {
           inherit src;
           pname = "git-toprepo";
-          version = "0.0.0";  # TODO: Use git-hash or similar.
+          version = self.rev or self.dirtyRev;
```

Comment:
On second though we don't want this. Setting the package version to the revision will force rebuild the package on every change.

Reply:
https://github.com/meroton/git-toprepo/blob/9b4bdd2bb4822b9161462844a411a3e07c90465b/src/main.rs#L485 is what is printed on `git-toprepo version`.

Reply:
Nice. I think I have an idea what we can do.